eahub/base/views.py

```diff
@@ -193,11 +192,6 @@ def get_private_profiles(user):
     return private_profiles_json
 
 
-class FaviconView(base.RedirectView):
-    def get_redirect_url(self):
-        return static.static("favicon.ico")
-
-
```

Comment:
I didn't understand the purpose of it, hence I removed it to check if it's going to change anything. So far nothing